getting-started-with-open-data-hub.adoc

```diff
@@ -56,6 +56,8 @@ include::modules/disabling-applications-connected-to-openshift-data-science.adoc
 include::modules/removing-disabled-applications-from-openshift-data-science.adoc[leveloffset=+2]
 endif::[]
 
+include::modules/configuring-ide.adoc[leveloffset=+1]
```

Comment:
The filename must match the module title. For a module name "Configuring your IDE", its corresponding filename should be: 

configuring-your-ide.adoc